templates/referent/nomination/referent.html.twig

```diff
@@ -1,7 +1,9 @@
 {% extends 'legislatives/layout.html.twig' %}
 
 {% block page_title referent.fullName %}
-{% block canonical_url url('our_referents_referent', {'slug': referent.slug}) %}
+{% block canonical_url url('our_referents_referent', { slug: referent.slug}) %}
+{% block meta_description referent.fullName ~ ' est le ou la référent(e) du département '~ referent.areaLabel ~'. Il ou elle répresente le mouvement de La République En Marche ! au sein de ce territoire et est en charge de la coordination avec les élus, les comités locaux et les représentants du mouvement au niveau national.' %}
```

Comment:
Wouldn't be more readable with interpolation in this case? (`#{}`)

Reply:
Yes nice idea, thx !